src/olympia/activity/tests/test_serializers.py

```diff
@@ -38,7 +38,7 @@ def test_basic(self):
         result = self.serialize()
 
         assert result['id'] == self.entry.pk
-        assert result['date'] == self.now.isoformat()
+        assert result['date'] == self.now.isoformat() + 'Z'
```

Comment:
oh we'll need moooooore tests, assuming we're exposing it here and there but maybe that's the only place?